### ISSUE TITLE
:bug: Since v2.4.3 any hub other than classic isn’t working correctly

### ISSUE THREAD

**Issue by TimboZero** [jurialmunkey/skin.arctic.fuse.2#348]
### Skin section

Other

### Current Behavior

I have hub 1101 as my customised widget only (no vertical or horizontal category menu, Detailed mode), since v2.4.3 it can no longer have its mode or name changed whilst it is set as Home, and it (hub 1101) has been removed from the hub list below. If I try to ‘workaround’ by selecting classic as home the hub 1101 reappears in hub list and I can edit it, but as soon as I set it as Home it becomes ‘empty’ homepage (and config but I can edit it within classic hub parameters/limitations such as having to have a category menu for example)

### Expected Behavior

When none classic hub is set as home it should still appear in hub list to be edited.
When Home is set to non-classic it should inherit that hubs config, as it did before 2.4.3 

### Steps To Reproduce

Make a non-classic hub (such as hub 1101) and customise it to your liking, my config was Detailed with no categories (hence no ‘menu’) and several widgets defined as rows (movies, tv shows, music, favourites, addons)

Set Hub as Home, and that hub will disappear from hub list

Exit skin config, allow skin variables to config your home and once done you’ll have an empty home.
(Note, the skin update displayed home correctly until I changed one of widgets from poster to landscape, reverting the change didn’t help)

### Checklist

- [X] I have made a separate issue for each bug that I am reporting.
- [X] I have searched the issues and this bug has not been reported.
- [X] I am using the most recent skin version.
- [X] I have confirmed that this bug does not occur in the default skin.

**Comment by jurialmunkey** [comment]
Yes, from 2.4.3 it forces categories on when using a hub as home. My thinking was to avoid users from locking themselves out of the settings screen but I can see the value in having a completely widgets only home screen so I will change it.

I just didn't really think anyone would be using a hub as home this way as the classic home screen already has a widgets only layout and there is already an option to set initial focus to widgets which also wraps the widgets on the side (instead of opening the menu) so that only back would open the side menu.

**Comment by jurialmunkey** [comment]
^^^ Will still force into advanced mode when selecting the hub to be home to try to avoid people making dumb mistakes but will allow the mode to be overridden to be basic/details.

I do have a big warning on the textbox so people can't say I didn't warn them!

**Comment by TimboZero** [comment]
I agree the warning is needed but…

on remote (assume equivalent on other platforms/controls) a long back press opens side menu (this maybe a platform/remote specific thing but could be a default skin setting like normal back press has?)

I would advocate for the idea that old method with a double ‘are you sure’ ‘do you have settings available’ kind of warning could be better.

**Comment by TimboZero** [comment]
> ^^^ Will still force into advanced mode when selecting the hub to be home to try to avoid people making dumb mistakes but will allow the mode to be overridden to be basic/details.
> 
> I do have a big warning on the textbox so people can't say I didn't warn them!

It also seems to me to essentially blank my widget only hub, I can’t change the mode (no setting to change custom hub once set as home and it’s removed from hub list) and can’t add widgets without adding a category (to remake my hub as home)

**Comment by TimboZero** [comment]
Apologies for crappy iPad pic

![image](https://github.com/user-attachments/assets/d07ab03b-793c-4df2-add3-5e94eab0cee1)

**Comment by TimboZero** [comment]
Thank you for the change in 2.4.6 allowing this type of config again, appreciate the change and your continued work on this project.

**Comment by iparr** [comment]
I accidentally set the home widget to a different hub and now all of my menus have totally disappeared. I can't navigate anywhere.

~~I have managed to delete the skin completely, but upon re-install I'm plunged into the same broken setup. How does one return things to how they were - or failing that - to sensible defaults?~~

How I fixed this:
1. ssh onto machine
2. `systemctl stop kodi`
3. `nano ~/.kodi/userdata/addon_data/skin.artic.fuse.2`
4. Delete the line `<setting id="hub.home.replacewindow" type="string"></setting>`
5. `systemctl start kodi`

I wish I could get the skin looking like in the screenshots!
